bookserver/crud.py

```diff
@@ -97,27 +94,23 @@ async def fetch_last_answer_table_entry(query_data: schemas.AssessmentRequest):
         )
         .order_by(tbl.timestamp.desc())
     )
-    async with async_session() as session:
```

Comment:
These queries were working just fine. I thought we had come to agreement on our Sqlalchemy strategy...  Why the changes??

Reply:
The more I'm using SQLAlchemy v1.4, the more I realize that the choice isn't exclusive. My thoughts:
- Defining tables ORM style is cleaner and more readable. Let's continue using that.
- Using columns ORM style (`tbl.div_id`) is cleaner and more readable than columns Core style (`tbl.c.div_id`). Let's continue using that.
- Most DB operations (queries, inserts, etc.) are very, very similar. For these, let's pick the style that meets our design goals.

So, my main point is: we can freely mix and match ORM vs. Core, picking the style that's best for the situation. In the case of a query, the options are:
- ORM: We `select(tbl)`, then get the results in a class. Since it's an ORM class, we then need to convert it to something else (a Pydantic class is probably easiest) so that accidental changes won't affect the database.
- Core: we `select(tbl.t)`, then get the results in a class-like object we can directly return. This seems slightly easier to me.

Reply:
At this point I think we have made things needlessly complicated.  Defining the model with ORM syntax, and creating objects with ORM syntax, then adding a metaclass to get the underlying `.__table__ ` so that we can return a Row instead of a Pydantic object.  I think this kind of complexity makes it hard for others to join in the project.

I'm going to be honest, I love Python because it is expressive and lets me get stuff done quickly.  I feel like we are perilously close to turning Python into Java with all of this validation and type hinting, and linting...  Every time I try to do a commit and can't because flake8 says I have an unused import I cringe and say a bad word.  

But I want us to be consistent, and I want us to have code that is accessible to newcomers who want to contribute, and I want the bookserver to be fast and super reliable.

So... I'm trying bend and see if I can get along with type hints and whatnot.   I'm trying to see FastAPI as a model project.

`crud.py` is still a work in progress, so I appreciate the experimentation.  But I'm also feeling a lot of urgency to have a decision made and get on with things.

1. I think it would be simpler for us to clearly choose Core or ORM and not some hybrid.
2. I agree that we want to avoid returning ORM objects and having open sessions and all of the behind the scenes automatic update that could lead to hard to debug issues.
3. We do not embed a lot of logic into our model objects.  The only place we really do that is in the dashboard and the performance there is horrible.  Largely because tons of stuff that could be done with better queries is done with Python loops. 
4. Most of ajax.py is very basic CRUD type operations.  No complex business logic to embed there.
5. I like the idea of being able to declare the kind of object each crud function is going to return for the sake of mypy but also because vscode pylance has good integration with pydantic and can do autocomplete.  sqlalchemy rows are just rows -- no autocompletion.

So, maybe I'm saying Core is the best way to go, but that I would still prefer to convert crud results to a pydantic class.  I think mypy will catch your whooops forgot to convert scenario very quickly!